docs/source/tutorial/commercial_paper.md

```diff
@@ -450,135 +357,167 @@ Feel free to examine other files in the `contract` directory to understand how
 the smart contract works, and read in detail how `papercontract.js` is
 designed in the smart contract [topic](../developapps/smartcontract.html).
 
-## Install contract
+## Deploy the smart contract to the channel
 
 Before `papercontract` can be invoked by applications, it must be installed onto
-the appropriate peer nodes in PaperNet.  MagnetoCorp and DigiBank administrators
-are able to install `papercontract` onto peers over which they respectively have
-authority.
+the appropriate peer nodes of the test network and then defined on the channel
+using the [Fabric chaincode lifecycle](../chaincode4noah.html). The Fabric chaincode
+lifecycle allows multiple organizations to agree to the parameters of a chaincode
```

Comment:
A better link would be directly to: [Fabric chaincode lifecycle] (/..chaincode4noah.html#chaincode-lifecycle).